packages/wasm-dpp/src/data_contract/data_contract.rs

```diff
@@ -153,7 +153,7 @@ impl DataContractWasm {
             .document_type_for_name(doc_type)
             .with_js_error()?;
 
-        let binary_paths_o = document_type.binary_paths();
+        let _unused_binary_paths_o = document_type.binary_paths();
```

Comment:
I think we just need to remove such unnecessary calls.

Reply:
probably better to do in another PR maybe? I wanted to keep diff / potential for objection low. Simply renaming is best to achieve that for now and get rid of warning. 

Although if it's thought best to remove; I could without too much effort

Reply:
remove